knowledge-base/common-readonly-properties

```diff
@@ -0,0 +1,44 @@
+---
+title: Error when using read-only model properties and editing
```

Comment:
Use title case. See the [capitalization rules](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).